pkg/limits/ingest_limits.go

```diff
@@ -198,14 +223,33 @@ func (s *IngestLimits) evictOldStreams(ctx context.Context) {
 			s.mtx.Lock()
 			cutoff := time.Now().Add(-s.cfg.WindowSize).UnixNano()
 			for tenant, streams := range s.metadata {
+				streamsBefore := len(streams)
+				evictedCount := 0
```

Comment:
IMO no need for `count` suffix, it's obvious these are counters. If you do want to keep a description in the variable name, we seem to use `num` here. `numEvicted` `numActive` etc.

Reply:
You could also do at the end of the for loop:

```go
numActive := len(streams) - numEvicted
```